src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -59,29 +74,68 @@ export const Chatbox: React.FC<ChatboxProps> = ({
     [messages, deleteMessage]
   );
 
+  const dropdownOptions = useMemo(
+    () =>
+      OPTIONS.map((option) => (
+        <Dropdown.Item
+          key={option.name}
```

Comment:
Are option names guaranteed to always be unique? Currently since it's an array I don't think they are. Though after changing things to use an object as @mike-lvov suggested in https://github.com/sparkletown/sparkle/pull/1343/files?diff=unified&w=1#r622457060 it should be.

Reply:
fixed in b1c8ac0